Xamarin.Forms.Core/PlatformConfiguration/iOSSpecific/Page.cs

```diff
@@ -60,10 +60,14 @@ public static UIStatusBarAnimation PreferredStatusBarUpdateAnimation(this IPlatf
 
 		public static readonly BindableProperty UseSafeAreaProperty = BindableProperty.Create("UseSafeArea", typeof(bool), typeof(Page), false, propertyChanged: (bindable, oldValue, newValue) =>
 		{
+			if (Device.RuntimePlatform != Device.iOS) return;
```

Comment:
if you ever need to change this PR for another reason, it'd be a good opportunity to move the propertychangedhandler into a real method, for clarity